### ISSUE TITLE
Use coursier for better perf

### ISSUE THREAD

**Issue by pauldraper** [johnynek/bazel-deps#152]
Dependening on your network latency and number of resolvers, bazel-deps can be excruciatingly slow on a lot of dependencies.

We could replace Aether with [coursier](https://github.com/coursier/coursier), which is resolves Maven/Ivy in parallel. (Plus, with built-in caching and a less archaic API.)

That's a significant change, but IDK how else to improve the resolution time from many minutes.

---

EDIT: Or maybe we can add concurrency to bazel-deps?

**Comment by pauldraper** [comment]
Already done last week :grinning: #145
